test/entropic/sinkhorn_barycenter.jl

```diff
@@ -1,6 +1,7 @@
 using OptimalTransport
 
 using Distances
+using ForwardDiff
```

Comment:
Is it actually needed?
```suggestion
```

Reply:
Nope - sorry this seems to be related to spillover from some other branch :/ I'll take a look through files to double check.